src/Uno.UI/UI/Xaml/Controls/SplitView/SplitView.iOS.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Animation;
+
+namespace Windows.UI.Xaml.Controls
+{
+	partial class SplitView
+	{
+		/*
+
+			About the PatchInvalidFinalState:
+				On iOS for some strange reasons, it may occures that the burger menu does not completes its open/close animation,
```

Comment:
"it may occures" doesn't work
```suggestion
				On iOS for some strange reasons, the burger menu may not complete its open/close animation,
```